app/javascript/components/App.js

```diff
@@ -21,22 +31,31 @@ const App = () => {
   };
 
   return (
-    <BrowserRouter>
-      <Navbar open={open} setOpen={setOpen} />
-      <div className="component">
-        <button type="button" className="menu btn" onClick={handleClick}>
-          <FiMenu />
-        </button>
-        <Routes>
-          <Route path="/" element={<Main />} />
-          <Route path="RESERVE_FORM" element={<ReserveForm user={user} />} />
+    <>
+      {login ? (
+        <>
+          <Navbar open={open} setOpen={setOpen} />
+          <div className="component">
+            <button type="button" className="menu btn" onClick={handleClick}>
+              <FiMenu />
+            </button>
+            <Routes>
+              <Route path="/" element={<Main />} />
+              <Route path="RESERVE_FORM" element={<ReserveForm user={user} />} />
           <Route path="ADD_ITEM" element={<AddItem />} />
-          <Route path="MY_RESERVATION" element={<Reservation />} />
-          <Route path="DETAILS" element={<Details />} />
-          <Route path="DELETE_ITEM" element={<DeleteItem />} />
+              <Route path="MY_RESERVATION" element={<Reservation />} />
+              <Route path="DETAILS" element={<Details />} />
+              <Route path="DELETE_ITEM" element={<DeleteItem />} />
+            </Routes>
+          </div>
+        </>
+      ) : (
+        <Routes>
+          <Route path="/" element={<Registration />} />
+          <Route path="login" element={<Login />} />
```

Comment:
Maybe if we capture this into a main screen with two buttons for login and registration would give a better use experience.